pxr/base/plug/testPlug.py

```diff
@@ -275,10 +275,12 @@ def test_ErrorCases(self):
         self.assertEqual(listener.numReceived, 0)
 
         # try to load an unloadable plugin
-        badPlugin = Plug.Registry().GetPluginForType('TestPlugUnloadable')
-        self.assertIsNotNone(badPlugin)
-        with self.assertRaises(RuntimeError):
-            badPlugin.Load()
+        # XXX: This test is broken on Windows ARM64, so is switched off
```

Comment:
Just expanded this comment a bit, feel free to modify if anything sounds off here.

```suggestion
        # XXX:
        # On Windows ARM64 loading TestPlugUnloadable (which intentionally uses
        # an undefined external symbol) unexpectedly crashes in the underlying
        # LoadLibrary call with an EXCEPTION_ACCESS_VIOLATION instead of
        # returning NULL as it does on x64. For now we just skip this test
        # on that platform. More discussion at:
        # https://github.com/PixarAnimationStudios/OpenUSD/pull/3430.
```

Reply:
I have now added this and tweaked it slightly